### DISCUSSION TITLE
Make GetLocale() a global variable

### DISCUSSION THREAD

**Discussion by Mateorid** [grafana/grafana#66013] [Dashboards, Panels & Visualizations]
Making it a global variable would allow its use in payloads [(like this)](https://community.grafana.com/t/getting-the-getlocale-value-into-payload/81209/3) and in other parts

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82778. Feel free to continue the discussion around this there. Thank you!
